client/odie/data/index.ts

```diff
@@ -15,6 +15,19 @@ export const setOdieStorage = ( key: OdieStorageKey, value: string ) => {
 	const storageKey = buildOdieStorageKey( key );
 	localStorage.setItem( storageKey, value );
 
+	// Duplicate the value to support_chat_id
+	if ( key === 'chat_id' ) {
```

Comment:
Could you explain why it's needed to duplicate the ID like this? Do we need to clear it here? https://github.com/Automattic/wp-calypso/blob/283611d87c7bfc21214a754a0ce8c86c1d578763/packages/help-center/src/components/help-center-content.tsx#L110C6-L110C6

Reply:
Sure! It turns out that as soon as we click on the buttons to go either to Email, Forum, or Chat, we clear the chat_id so the next time the user goes to the chat, the conversation is empty. That way we can have the "last" chat_id always available, even after clearing it due to contacting user support.

Reply:
I've renamed it to better explain the intention